### ISSUE TITLE
SlevomatCodingStandard.Functions.StrictCall.NonStrictComparison false positive

### ISSUE THREAD

**Issue by paranoiq** [slevomat/coding-standard#1120]
the rule reports `Strict parameter should be set to true in in_array() call.` for following code:

```
$x = in_array($value, Type::listTypes(), Type::STRICT);
```

where class constant is used instead of `true` for clarity (from the times when PHPStorm did not show parameter names on calls)

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/79386c1662052c5bb9be6828ef3b31e270565847
